guides/common/modules/proc_migrating-project-or-smartproxy-using-backup.adoc

```diff
@@ -0,0 +1,43 @@
+[id="migrating-{project-context}-or-proxy-using-backup_{context}"]
+= Migrating {Project} or {SmartProxy} to {EL-abbr}{nbsp}9
+
+You can migrate your existing {ProjectServer} and {SmartProxyServer} from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+ifdef::satellite[]
+The migration involves creating a backup of the existing {ProjectServer} and {SmartProxyServer}, which you then restore or clone on the new {EL}{nbsp}9 system.
+[NOTE]
+====
+You can use cloning for {ProjectServer} backups but not for {SmartProxyServer} backups.
+====
+endif::[]
+ifndef::satellite[]
+The migration involves creating a backup of the existing {ProjectServer} and {SmartProxyServer}, which you then restore on the new {EL}{nbsp}9 system.
+endif::[]
+
+.Procedure
+. Perform a full backup of your {Project} or {SmartProxy}.
+This is the source {EL}{nbsp}8 server that you are migrating.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin#Performing_a_Full_Backup_admin[Performing a full backup of {ProjectServer} or {SmartProxyServer}] in _{AdministeringDocTitle}_.
+. Deploy a system with {EL}{nbsp}9.
+This is the target server.
+Install {ProjectServer} or {SmartProxyServer} on the target {EL}{nbsp}9 system:
+ifeval::[ "{mode}" == "connected" ]
+** To install connected {ProjectServer}, see {InstallingServerDocURL}[_{InstallingServerDocTitle}_].
+endif::[]
+ifeval::[ "{mode}" == "disconnected" ]
+** To install disconnected {ProjectServer}, see {InstallingServerDisconnectedDocURL}[_{InstallingServerDisconnectedDocTitle}_].
+endif::[]
```

Comment:
```suggestion
ifeval::[ "{mode}" == "connected" ]
** To install {ProjectServer}, see {InstallingServerDocURL}[_{InstallingServerDocTitle}_].
endif::[]
ifeval::[ "{mode}" == "disconnected" ]
** To install {ProjectServer}, see {InstallingServerDisconnectedDocURL}[_{InstallingServerDisconnectedDocTitle}_].
endif::[]
```
I strongly suggest removing those keywords. Two reasons: 1) in upstream, there's only one ProjectServer and that is connected, and 2) the keywords appear in the DocTitles where appropriate.